### DISCUSSION TITLE
Integrating Auth with Jetstream/Fortify

### DISCUSSION THREAD

**Discussion by Kayrah87** [area17/twill#1643] [Q&A]
Hi, I've searched high and low for an answer to the question but with limited answers.

I need to integrate the Twill admin panel with a fully implemented Jetstream-based app. The login is handled by Fortify as per Jetstream norms.
I use Inertia and VueJS for the front end and have built a good static site already with a member dashboard.

I now need to add CMS capability - hence the Admin page and the cool functionality of Twill. 

I have changed the `twill_users` table in the config to `users` according to the docs, and I have the admin page and domain set up as per the docs: 
```
APP_URL=domain.test
ADMIN_APP_URL=domain.test
ADMIN_APP_PATH=admin
SESSION_DOMAIN=domain.test
```
However, I am still getting the Twill login page whenever I go to `/admin`.

My last resort will be to switch to the Twill login page and then trigger a `Auth::login($twill_user)` via middleware or helper. But I would like to keep the Fortify login page I have if possible.

**Comment by haringsrob** [comment]
Hey @Kayrah87,

What I did in the past was during login also run `Auth::guard('twill_users')->login($user);`

This way you do not have to change much and the user will be logged into the twill guard as well.
